pyk/src/tests/unit/ktool/test_krun.py

```diff
@@ -23,6 +23,7 @@
         ('search_final', False),
         ('no_pattern', False),
         ('debugger', False),
+        ('proof_hint', False),
```

Comment:
Please add an integration test for checking the proof hints output for some execution.